consumer/consumer.go

```
@@ -39,6 +42,19 @@ func (c *consumer) start(ctx context.Context) {
 			// and save the tick data to the data store
+			fetches := c.client.PollFetches(ctx)
+			if errs := fetches.Errors(); len(errs) > 0 {
+				panic(fmt.Sprint(errs))
+			}
+
+			iter := fetches.RecordIter()
+			for !iter.Done() {
+				record := iter.Next()
+				fmt.Println(string(record.Value), "from an iterator!")
+				tick := data.Tick{}
+				json.Unmarshal(record.Value, &tick)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)